sdk/communication/azure-communication-sms/README.md

```diff
@@ -92,8 +92,10 @@ There are two different forms of authentication to use the Azure Communication S
 ### Send a 1:1 SMS Message
 Use the `send` or `sendWithResponse` function to send a SMS message to a single phone number.
 
-<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L68-L75 -->
+<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L119-L128 -->
 ```java
+SmsClient smsClient = createSmsClientUsingAzureKeyCredential();
```

Comment:
I think it's sufficient that we have the examples above that already show how to create a client.
I think it's redundant to put this function here, especially since it's a helper function that's not visible from the readme.